samples/ibm-samples/watson/watson_train_serve_pipeline.py

```diff
@@ -49,29 +46,35 @@ def kfp_wml_pipeline(
 ):
     # op1 - this operation will create the credentials as secrets to be used by other operations
     get_configuration = configuration_op(
-                   token = GITHUB_TOKEN,
-                   url = CONFIG_FILE_URL,
-                   name = secret_name
+                   token=GITHUB_TOKEN,
+                   url=CONFIG_FILE_URL,
+                   name=secret_name
     )
-    
+
     # op2 - this operation trains the model with the model codes and data saved in the cloud object store
     wml_train = train_op(
-                   get_configuration.output,
-                   train_code,
-                   execution_command
+                   config=get_configuration.output,
+                   train_code=train_code,
+                   execution_command=execution_command,
+                   framework=framework,
+                   framework_version=framework_version,
+                   runtime=runtime,
+                   runtime_version=runtime_version,
+                   run_definition=run_definition,
+                   run_name=run_name
```

Comment:
since we can just go with the variable values, maybe we can remove duplication here

Reply:
I removed duplications on the last 2 components since they have few arguments. For the train component, I decided to keep using keyword arguments since it has a long list of args and users might mistakenly put them in the wrong order.

Reply:
My experience is that this makes it less error-prone and also makes it easier for users.
Without explicit names, if you add a new parameter, the calls may become invalid since the wrong arguments will be passed.
Also this makes it easier since the users see the parameter names which can be different from argument names, especially in cases where arguments are constant.

Reply:
Yeah we discussed internally extensively, specially if there are large number of params. Maybe it would behoove to make it a standard, specially on our pipelines. i am not a fan of mix and match in general